.github/workflows/interchaintest-e2e.yml

```
@@ -55,16 +55,18 @@ jobs:
         test:
-          # - "e2e-basic"
+          - "e2e-basic"
+          # - "e2e-pfm"
+          # - "e2e-polytone"
           # - "e2e-upgrade"
       fail-fast: false
 
     steps:
       - name: Set up Go ${{ env.GO_VERSION }}
-        uses: actions/setup-go@v5
+        uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)